### DISCUSSION TITLE
keyball61の左手上一列の入力がおかしい

### DISCUSSION THREAD

**Discussion by yusenran** [Yowkees/keyball#502] [Q&A]
左手用基板を右手用基板に接続すると、「3455555555555...」と入力し続けられます。別の入力を行うと止まりますが、以降3,4,5の3つキーは反応しません。
また、1キーを入力すると「1qa」、2キーを入力すると「2ws」と同じ列の他のキーの入力も一緒にされてしまいます。
他のキーについては正常に入力できます。

![無題4_20240316023546 (1).png](https://github.com/Yowkees/keyball/assets/63898355/45f78901-2df1-4459-beab-5e89997d6ca3)

![IMG20240316022649.jpg](https://github.com/Yowkees/keyball/assets/63898355/e9b271e6-243d-4dff-b8c2-e07b7d236e22)

画像から具体的にどこに問題がありそうかもし分かれば教えていただけると幸いです。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

添付させていただいた写真の赤い丸で示した端子のはんだ付けがうまく馴染んでいないように見えますので、こちらのはんだ付けをやり直してみてください。
![313286289-e9b271e6-243d-4dff-b8c2-e07b7d236e22](https://github.com/Yowkees/keyball/assets/143698349/59de7aca-c0cc-4ed3-ab44-16bed880c3ef)

改善しない場合、ProMicroを左右で入れ替えるとどのようになるかお教えいただけますでしょうか。

**Reply by yusenran** [reply]
ありがとうございます。
ProMicroを左右で入れ替えたところ、現象がProMicroについてくることが分かりました。
半田は何度かやり直してみたのですが改善しないため、ProMicroごと新しいものに差し替えてみようかと思います。

**Reply by Ryo624** [reply]
ご対応ありがとうございます。

ProMicroの入れ替えで症状が逆転されるとのことですので、たしかにProMicroを新しいものに差し替えたほうがいいかもしれませんね。
白銀ラボのウェブショップからピンヘッダ等の部品も販売しておりますので、必要な場合はお問い合わせフォームからご連絡ください。
